src/clients/firstboot_language_keyboard.rb

```diff
@@ -258,7 +261,7 @@ def SetLanguageIfChanged(ret)
 
         # install language dependent packages now
         # Language::PackagesModified () does not work here as _on_entry variables are not set
-        if @language != Language.ReadLocaleConfLanguage
+        if @language != (Language.ReadLocaleConfLanguage || default_language)
```

Comment:
What happens when the current language is not the default one (i.e., English US), the user selects English US and reading locales fails? I guess that English US would not be installed.

Reply:
After discussing it with @joseivanlopez I agree that is better to check that the translation package is installed even for the default language or otherwise it could be ommited. Thus, I will close this PR and the related one.